### ISSUE TITLE
Combobox Not Announcing Selected State (NVDA)

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1202]
Auditor Issue: https://axeauditor.dequecloud.com/test-run/87ff92fc-25f1-11ee-a855-1b2298d1b894/issue/98b155e6-512e-11ee-b2ae-1b13575e65cf?sortField=ordinal&sortDir=asc&filter%5Brule%5D=manual&filter%5Bpage_number%5D=49&row=0

Best practice: The selected state of the element is not announced by default.

Element description: The selected state of the options is not announced to the screen reader users in the default state.
Element location: Located below the "Examples" heading

Note: It is applicable to all the Combo boxes.

Additional Information: Raised as a best practice as aria-selected="true" is provided and on "insert+tab", the screen reader announces the selected state.
However, by default when the focus goes to the element, the selected state is not announced.

**Comment by scurker** [comment]
@VaniChinnam This appears to be happening in NVDA, was this the case for other AT as well?

**Comment by scurker** [comment]
This appears to be a somewhat conscious decision on NVDA's part: https://github.com/nvaccess/nvda/issues/12893

> Only announcing one of the states (selected vs not selected) reduces verbosity, the other state is implied.
When considering this behavior in the context of a standard listbox (no multi select) this makes a lot more sense, the item focused IS selected, the user doesn't have to hear selected / vs not selected at all.

When testing this with NVDA "not selected" is announced for the `aria-selected="false"` options. I'm not sure what else we can do since NVDA has taken the stance to not redundantly announce selected items.
